examples/validate_relative_binding.py

```diff
@@ -45,6 +45,26 @@
 from md import builders, minimizer
 from rdkit.Chem import rdFMCS
 
+
+def cache_wrapper(cache_path: str, f):
```

Comment:
At first glance I thought `f` was a file object.

Naming suggestions:
* retain single letter name, add type hint `(cache_path: str, f)` -> `(cache_path: str, f: callable)`
* expand single letter name, so type hint would be redundant `(cache_path: str, f)` -> `(cache_path: str, fxn)`